tests/dummy/app/templates/docs/usage.md

```diff
@@ -17,23 +17,22 @@
   @enable={{this.datesToEnable}}
   @enableSeconds={{false}}
   @enableTime={{true}}
-  @flatpickrRef={{this.flatpickrRef}}   {{!-- two-way binding --}}
-  @getFlatpickrRef={{action (mut this.flatpickrRef)}}  {{!-- via action (prefered) --}}
+  @getFlatpickrRef={{this.setFlatpickrRef}}  {{!-- via action (prefered) --}}
   @hourIncrement={{1}}
   @inline={{false}}
   @locale="ru"
-  @maxDate={{maxDate}}
-  @minDate={{minDate}}
+  @maxDate={{this.maxDate}}
+  @minDate={{this.minDate}}
   @minuteIncrement={{5}}
   @mode="single"
   @nextArrow=">"
   @noCalendar={{false}}
-  @onChange={{action (mut dateValues)}} {{!-- Required Option --}}
-  @onClose={{action "doSomeStuffOnClose"}}
-  @onOpen={{action "doSomeStuffOnOpen"}}
-  @onReady={{action "doSomeStuffOnReady"}}
+  @onChange={{this.onChange}} {{!-- Required Option --}}
+  @onClose={{this.onClose}}
+  @onOpen={{this.onOpen}}
+  @onReady={{this.onReady}}
   @parseDate={{false}}
-  @placeholder="Choose a Date"
+  placeholder="Choose a Date"
```

Comment:
Isn't placeholder a flatpickr option, not the HTML `placeholder`?

Reply:
I looked through the flatpickr [options](https://flatpickr.js.org/options/) and there was no placeholder option.

Reply:
Ah sorry, my mistake! This is fine then 👍